docs/src/reference/4_linesearch.md

```diff
@@ -1,7 +1,7 @@
 # Line search and step size settings
 
 The step size dictates how far one traverses along a local descent direction.
-More specifically, the step size $gamma_t$ is used at each iteration to determine
+More specifically, the step size $\gamma_t$ is used at each iteration to determine
```

Comment:
when the preview is up, can you check that this renders correctly?

Reply:
Yeah I will do that